recipes/raspa3/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "raspa3" %}
+{% set version = "v3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/iRASPA/RASPA3/archive/{{ version }}.tar.gz
+  sha256: cfaf680615161b985261359410f0b59504efc392f4a1be54de7c48bc0b0d360a
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang-tools
+    - llvm-tools
+    - cmake
+    - ninja
+    - lld
+    - libcxx-devel  # [not win]
+  host:
+    - libcxx
+    - libcxxabi  # [linux]
+    - hdf5
+    - lapack
+    - blas
+    - libopenblas
+    - zlib
+  run:
+    - libcxx
+    - libcxxabi  # [linux]
+    - openssl
+    - libcurl
+    - libaec
```

Comment:
```suggestion
```
See my other comment below as to why these 3 libraries shouldn't be needed here.